app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -33,11 +33,7 @@ class LeaseUpApplicationsPage extends React.Component {
       loading: false
     }
   }
-
-  constructor (props) {
-    super(props)
-    this.eagerPagination = new EagerPagination(ROWS_PER_PAGE, SERVER_PAGE_SIZE)
-  }
+  eagerPagination = new EagerPagination(ROWS_PER_PAGE, SERVER_PAGE_SIZE)
```

Comment:
does this need to have a const attached to it?

Reply:
No this is technically still a class component so adding as a property of the class like this is the same as `this.eagerPagination`